packages/upload-api/src/access/authorize.js

```diff
@@ -81,7 +89,20 @@ export const authorize = async ({ capability }, ctx) => {
     url,
   })
 
-  return {
-    ok: {},
+  const ok = Server.ok({
+    // let client know when the confirmation will expire
+    expiration: confirmation.expiration * 1000,
```

Comment:
can we make this an ISO8601 datetime string of exactly when expiration will happen instead of a relative expiry length.?

Reply:
I think right now this `expiration` is number of milliseconds since epoch. would much prefer something more standard

Reply:
Can you give some rational why choosing datetime string would be a better choice ? I chose to use same name and format as UCAN (and ucan receipts) and would find having it in different format in different places inconsistent and more strange.

That said `* 1000` was a mistake and I should really do that and turn it into `Date` instance in the `PendingRequest` just like you've suggested.

Reply:
> I chose to use same name and format as UCAN (and ucan receipts) and would find having it in different format in different places inconsistent and more strange.

that's a good reason to do it that way then, consistent with other stuff.